api/src/main/java/org/openmrs/api/db/SerializedObject.java

```diff
@@ -28,6 +31,9 @@
 @Entity
 @Table(name = "serialized_object")
 @Audited
+@AttributeOverrides(value = {
+	@AttributeOverride(name = "description", column = @Column(name = "description", length = 5000))
```

Comment:
Shouldn't this also be text?

Reply:
Oh wait, why should a description be that large?

Reply:
create table serialized_object
(
    serialized_object_id int auto_increment
        primary key,
    name                 varchar(255)         not null,
    description          varchar(5000)        null,
    type                 varchar(255)         not null,
    subtype              varchar(255)         not null,
    serialization_class  varchar(255)         not null,
    serialized_data      mediumtext           not null,
    date_created         datetime             not null,
    creator              int                  not null,
    date_changed         datetime             null,
    changed_by           int                  null,
    retired              tinyint(1) default 0 not null,
    date_retired         datetime             null,
    retired_by           int                  null,
    retire_reason        varchar(1000)        null,
    uuid                 char(38)             not null,
    constraint uuid
        unique (uuid),
    constraint serialized_object_changed_by
        foreign key (changed_by) references users (user_id),
    constraint serialized_object_creator
        foreign key (creator) references users (user_id),
    constraint serialized_object_retired_by
        foreign key (retired_by) references users (user_id)
);

DDL for this table looks like this. What do you suggest?

Reply:
Looks like that was a typo 😊

Reply:
should it be `text` or `varchar` ?